components/hab/mac/mac-build.sh

```diff
@@ -30,33 +30,34 @@ install_if_missing() {
     local formula="$pkg"
   fi
 
-  echo $(sudo -u ${SUDO_USER} brew list --versions ${pkg})
+  sudo -u "${SUDO_USER}" brew list --versions "${pkg}"
 
-  if [[ $(sudo -u ${SUDO_USER} brew list --versions $pkg | wc -l) -eq 0 ]]; then
+  if [[ $(sudo -u "${SUDO_USER}" brew list --versions "$pkg" | wc -l) -eq 0 ]]; then
     info "Installing missing Homebrew package $formula"
-    sudo -u $SUDO_USER brew install $formula
+    sudo -u "$SUDO_USER" brew install "$formula"
   fi
 }
 
-if (( $EUID != 0 )); then
+if (( EUID != 0 )); then
```

Comment:
This comes up in a number of places: https://github.com/koalaman/shellcheck/wiki/SC2004#-is-unnecessary-on-arithmetic-variables